views/scss/item-creator.scss

```diff
@@ -3,6 +3,9 @@
 
 //colors used accross the item creator
 $itemCreatorSideBarBgColor: mix(#fff, $grey, 85%);
+$itemCreatorSideBarHeaderColor: mix(#fff, $websiteBorder, 70%);
+$itemCreatorWidgetElementBorderColor: mix(#fff, $info, 20%);
+$itemCreatorWidgetElementBackgroundColor: mix(#fff, $info, 90%);
 
```

Comment:
Could you please clarify what are sideBar and Element ?
If those colors are used for styling the mini toolbar then it might be better to duplicate them in different variable with explicit names:
`$lightBlue: mix(#fff, $info, 90%);`
`$itemCreatorWidgetElementBg: $lightBlue;`
`$itemCreatorActionBtnBg: $lightBlue;`

The fact that they use the same color does not mean that these elements will have their color changed at the same time and for the same new colors.